### PR TITLE
[deps] Added support for Django 3.0

### LINKED ISSUE #79: [deps] Update dependencies (django 3.0)
Add support for django 3.0 and update dependencies.

Drop support for python 3.5, add 3.6 and 3.7 to the travis build.

### PR DESCRIPTION
**Pull request by TheOneAboveAllTitan**
Added support for Django 3.0
Bumped openwsip-user to 0.2.0
Fixes #79

### CODE REVIEW

File: openwisp_radius/tests/test_api.py [hunk 1]
```diff
@@ -180,7 +180,7 @@ def test_api_password_change(self):
         }
         response = client.post(password_change_url, data=new_password_payload)
         self.assertEqual(response.status_code, 400)
-        self.assertIn("The two password fields didn't match.", response.data['new_password2'])
+        self.assertIn("The two password fields didn’t match.", response.data['new_password2'])
```
Comment (nemesisdesign) [inline openwisp_radius/tests/test_api.py]: try `str(response.data['new_password2'])`

Comment (TheOneAboveAllTitan) [reply openwisp_radius/tests/test_api.py]: This one is passing on my machine 
`test_radius_group_delete_default_by_non_superuser` is the one which is failing after installing `django-freeradius` again. Local environment was not updated with GitHub repository.

Comment (TheOneAboveAllTitan) [reply openwisp_radius/tests/test_api.py]: In last line, shouldn't it be `confirm_response` instead of `response` ?

Comment (TheOneAboveAllTitan) [reply openwisp_radius/tests/test_api.py]: Django 2.2 uses `'` for contractions while Django 3 uses `’` for same. This is causing test to fail.

**Comment by TheOneAboveAllTitan** [comment]
Have a look here https://github.com/openwisp/django-freeradius/issues/271

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
 django-freeradius<0.2.0
-openwisp-users>=0.1.12,<0.2
+openwisp-users>=0.1.12,<0.2.1
```
Comment (nemesisdesign) [inline requirements.txt]: This should have been `openwisp-users>=0.1.12,<0.2.1` but I will fix it.

**Pull request merged**